### ISSUE TITLE
Mistaken start of schedule

### ISSUE THREAD

**Issue by longhotsummer** [Code4SA/indigo#34]
Indigo thinks the following starts a new schedule:

```
(2) Schedule 1 of the Public Enterprises Governance Act , 2006 (Act 2 of 2006), is amended by the insertion after item 52 of the following item:
"53 Communications Regulatory Authority of Namibia Communications Act , 2009 (Act 8 of 2009 )"
[Subsec (2) amended by sec 15 of Act 8 of 2015.]
```

Change it to 
```
(2) The Schedule ...
```

fixes it.

**Comment by longhotsummer** [comment]
The reason this happens is the same reason that ``(2) (a) blah blah`` works, which makes working around it tricky. If we can ensure that ``(2) (a)`` isn't standard, then we can disallow EOLs. We'll need to look at how we accumulate content in subsections (including tables).

**Comment by longhotsummer** [comment]
Fixed in df5e217 with slaw 0.13
